### DISCUSSION TITLE
Is deployment for payload custom website series working?

### DISCUSSION THREAD

**Discussion by madman3** [payloadcms/payload#1067] [Q&A] [answered]
I followed the youtube video by @jmikrut and tried deploying [custom website series](https://github.com/payloadcms/custom-website-series) to aws. Unfortunately, I can't get it working because payload seems to have an issue communicating with next js app. Initially, I thought it was to do with a typescript error and that wasn't the case. But, I believe it may be a configuration issue on apache.

I've opened [an issue](https://github.com/payloadcms/custom-website-series/issues/4) on the custom website series almost 2 months ago. I've tried posting it also on stackoverflow without any luck. I saw that the payload discord server is also not very active on community help. I have even put out a job on freelancing sites to get this fixed and it hasn't been easy to find someone to fix it yet.

Can anyone please help me with what I maybe getting wrong? Has anyone got the deployment of custom website series working? I've been struggling with this one for weeks and it would've been awesome to have an active community where builders on payload could help each other out.

**Comment by jmikrut** [comment, marked as answer]
Hey @madman3 — yes, deployment works for that repo and you can see it deployed in production with the live site that reflects the custom website series here:

https://metricstructures.com

This is likely something to do with your environment. What you should do is try and determine _**why**_ that request to your API is failing. You should look at the Apache logs. Your request is failing at the Apache level, and the errors that you are seeing are because your Apache server is sending back HTML instead of sending back the response from the Payload API - meaning that your Apache server is not successfully proxying the request to your Node server. This is solely related to your server, unfortunately.

Typically we would recommend running Node behind an nginx reverse proxy. There are lots of tutorials online that show how to do that but I am not as well versed with Apache.

One other thing you could do is to try and _**directly hit your Payload API**_ in your browser - like, go straight here:

```
${process.env.NEXT_PUBLIC_SERVER_URL}/api/studies
```

In your browser. See if you can load it directly. I bet you will see the Apache error page.

Give it a shot!

**Reply by madman3** [reply]
Thank you so much, @jmikrut. This was super helpful! I love what you guys are building.
